tsconfig.base.json

```diff
@@ -15,6 +15,9 @@
     "skipDefaultLibCheck": true,
     "baseUrl": ".",
     "paths": {
+      "@elewa-group/features/components/cards": [
+        "libs/features/components/cards/src/index.ts"
+      ],
       "@elewa-group/pages/elewa/about-us": [
         "libs/pages/elewa/about-us/src/index.ts"
       ],
```

Comment:
where are all the other paths, please sync your fork

Reply:
I have been syncing my fork all through, or you meant I do it now?